frontend/src/pages/modelServing/screens/projects/NIMServiceModal/ManageNIMServingModal.tsx

```diff
@@ -147,8 +148,11 @@ const ManageNIMServingModal: React.FC<ManageNIMServingModalProps> = ({
   );
 
   const isStorageClassesAvailable = useIsAreaAvailable(SupportedArea.STORAGE_CLASSES).status;
-  const [defaultSc] = useAdminDefaultStorageClass();
-  const defaultStorageClassName = defaultSc?.metadata.name || '';
+  const odhDefaultScResult = useAdminDefaultStorageClass();
+  const openshiftDefaultSc = useOpenshiftDefaultStorageClass();
+  const odhDefaultSc = odhDefaultScResult[0];
+  const defaultSc = odhDefaultSc || openshiftDefaultSc;
+  const defaultStorageClassName = defaultSc ? defaultSc.metadata.name || '' : '';
```

Comment:
Very minor nitpick but the ternary seems a little redundant 
```suggestion
  const defaultStorageClassName = defaultSc?.metadata.name || '';
```

(Also technically you only have to fall back to the empty string when it's nullish so i think a `??` is more accurate to have)
```suggestion
  const defaultStorageClassName = defaultSc?.metadata.name ?? '';
```